login.html

```diff
@@ -60,6 +66,27 @@
         <img src="./assets/images/blob.png" id="blob">
     </div>
 
+    <!-- FOOTER -->
+    <div class="footer">
+        <div class="row mx-0">
+          <div class="col-md-6 mx-auto">
+            <div class="row mx-0 align-items-center">
+              <div class="col-sm-6 d-flex my-3 py-auto flex-column align-items-center order-sm-1 order-2">
+                <img src="./assets/images/logo.png" class='navLogo ' alt="..">
```

Comment:
Image size can be minimized

Reply:
How small do we need it?

Reply:
56px is fine

Reply:
I tried it and it looks too small. Do you want me to stick with 56?

Reply:
According to me, 70px looks fine. What do you say?

Reply:
Sure, anything that you think is fine.